cookbook/activity_manager/add_project_remover.rst

```diff
@@ -0,0 +1,150 @@
+How To Remove A Project On A Custom Catalog Update?
+===================================================
+
+.. _reference: ../../reference/activity_manager/catalog_impact.html
+
+Several catalog updates can impact projects integrity. In this case we remove impacted projects. Akeneo PIM already
+manages its own catalog updates. The list is available in Activity Manager reference_ part.
+
+How does It Work?
+_________________
+
+A subscriber handles catalog structure updates listening to `Akeneo\Component\StorageUtils\StorageEvents::POST_SAVE`
+and `Akeneo\Component\StorageUtils\StorageEvents::PRE_REMOVE` on all entities and dispatching them to the
+`PimEnterprise\Component\ActivityManager\Remover\ChainedProjectRemover`. This chained project remover contains all
+project removers and guesses which one has the responsibility to handle both entity and action it received.
+
+A `ProjectRemover` has the responsibility to find and remove all projects impacted by entity and action pair.
+
+How To Add A Custom Project Remover?
+____________________________________
+
+Reference data will be a good example to demonstrate how to add a `ProjectRemover`. We will take the example of Color
+reference data.
```

Comment:
of the Color reference data which is provided by default. See pim-community-dev repository `app/config.yml` and `app/AppKernel.php` to enable it.